ci/requirements/py36.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
   - distributed
   - h5py
   - h5netcdf
-  - matplotlib
+  - matplotlib=3.0  # for https://github.com/barronh/pseudonetcdf/issues/69
```

Comment:
Why only py36.yml? What about the other requirements files?
Also, will this restriction appear in the conda recipe?

Reply:
This is just to ensure that we have at least one test job testing xarray with psuedonetcdf, which doesn't import in matplotlib 3.1 due to the linked issue. Xarray itself works perfectly fine on matplotlib 3.1.